src/jekyll/_includes/page-structure/navigation.liquid

```diff
@@ -17,7 +17,7 @@ We don't have a content nav for the root of /web/ so for this we just skip
         {{ contentnavTocItem.title }}
       </a>
       {% endif %}
-      {% if contentnavTocItem.hasSubNav %}
+      {% if contentnavTocItem.hasSubNav == true and contentnavTocItem.isInCurrentSection == true %}
```

Comment:
This sets up the nav so that only the subnav items for the current section are visible, other areas just show their top level nav. The nav was longer than I liked without it.